workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/MonoBehaviourActivationManager.cs

```diff
@@ -271,7 +271,7 @@ private void RunWithExceptionHandling(Action action)
             catch (Exception e)
             {
                 // Log the exception but do not rethrow it.
-                logger.HandleLog(LogType.Exception, new LogEvent().WithException(e));
+                logger.HandleLog(LogType.Exception, new LogEvent("Caught exception").WithException(e));
```

Comment:
`Caught exception while executing an action` 

If we know these are only used in callbacks in the Monobehaviour workflow we could even be more explicit?

Reply:
I just didn't want to make any assumptions about what the caller is doing. Also "while executing action" seemed redundant, the user will see a stack trace anyway.

Reply:
The content of the action could throw, but these actions are all callbacks registered against a Reader/Writer right now I think